frontend/components/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "vue-strap",
+  "version": "2.14.1",
+  "description": "Bootstrap components built with Vue.js",
+  "main": "dist/vue-strap.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MarkBind/markbind.git"
+  },
+  "homepage": "https://markbind.org",
+  "license": "MIT",
+  "dependencies": {
+    "bootstrap": "^3.3.7",
+    "jquery": "^3.3.1",
+    "terser-webpack-plugin": "^2.3.5",
+    "vue": "^2.0.0",
+    "vue-loader": "^15.9.1",
+    "webpack": "^4.42.1",
+    "webpack-merge": "^4.2.2"
+  },
+  "files": [
+    "LICENSE",
+    "README.md",
+    "CHANGELOG.md",
+    "src",
+    "dist"
+  ],
+  "keywords": [
+    "bootstrap",
+    "vue-bootstrap",
+    "vue-component",
+    "vue",
+    "markbind"
+  ],
+  "devDependencies": {
+    "@babel/core": "^7.9.0",
+    "@babel/plugin-transform-runtime": "^7.9.0",
+    "@babel/preset-env": "^7.9.0",
+    "@babel/runtime": "^7.9.2",
+    "@vue/test-utils": "^1.0.0-beta.33",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "^25.4.0",
+    "babel-loader": "^8.1.0",
+    "css-loader": "^3.4.2",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-vue": "^6.2.2",
+    "expose-loader": "^0.7.5",
+    "jest": "^25.4.0",
+    "jest-serializer-vue": "^2.0.2",
+    "json-loader": "^0.5.7",
+    "less": "^3.11.1",
+    "less-loader": "^5.0.0",
+    "node-sass": "^4.13.1",
+    "sass-loader": "^8.0.2",
+    "style-loader": "^1.1.3",
+    "vue-hot-reload-api": "^2.3.4",
+    "vue-jest": "^3.0.5",
+    "vue-template-compiler": "^2.6.11",
+    "webpack-cli": "^3.3.11",
+    "webpack-dev-server": "^3.10.3"
+  },
+  "scripts": {
```

Comment:
Let's further adjust the order based on https://docs.npmjs.com/files/package.json.

```diff
  "name": "vue-strap",
  "version": "2.14.1",
  "description": "Bootstrap components built with Vue.js",
+ "keywords": [
+ "homepage": "https://markbind.org",
+ "license": "MIT",
+ "files": [
  "main": "dist/vue-strap.js",
  "repository": {
- "homepage": "https://markbind.org",
- "license": "MIT",
+ "scripts": {
  "dependencies": {
- "files": [
- "keywords": [
  "devDependencies": {
- "scripts": {
```

Reply:
will do the ones for `markbind/markbind-cli` in a seperate PR as well